testing/rippled/APKBUILD

```diff
@@ -30,9 +30,7 @@ package() {
 	cd "$builddir"
 	install -D -m644 LICENSE "${pkgdir}/usr/share/licenses/$pkgname/LICENSE"
 	install -D build/rippled "${pkgdir}/usr/bin/rippled"
-	install -D -m644 doc/rippled-example.cfg "${pkgdir}/etc/$pkgname/rippled.cfg"
-	install -D -m644 doc/validators-example.txt "${pkgdir}/etc/$pkgname/validators.txt"
```

Comment:
I this files became optional? I can't find answer in https://github.com/ripple/rippled/releases/tag/1.0.0

Reply:
Ah, right.... they just moved it to another directory. Sorry!